### DISCUSSION TITLE
Slow price reflection of new listing

### DISCUSSION THREAD

**Discussion by achie27** [amzn/selling-partner-api-models#3395] [General]
Does the price take ~2 hours to show/reflect on creating a new listing (with [`PutListingsItem`](https://developer-docs.amazon.com/sp-api/docs/listings-items-api-v2021-08-01-reference#listingsitemputrequest)) for you folks as well? It just shows `0.00` on the Seller Central dashboard meanwhile. 
I checked out a newly created listing (using [`GetListingsItem`](https://developer-docs.amazon.com/sp-api/docs/listings-items-api-v2021-08-01-reference#get-listings2021-08-01itemsselleridsku)) and noticed - 
- `purchasable_offer` in `attributes` is present with correct `value_with_tax`
- The top-level `offers` array has `price` set to 0

The correct price does show up eventually but it's incredibly slow.

Here's a sample request body I'm using -
```
{
  "attributes": {
    "condition_type": [
      {
        "value": "new_new",
        "marketplace_id": "ATVPDKIKX0DER"
      }
    ],
    "fulfillment_availability": [
      {
        "fulfillment_channel_code": "DEFAULT",
        "quantity": 10,
        "marketplace_id": "ATVPDKIKX0DER"
      }
    ],
    "merchant_shipping_group": [
      {
        "value": "legacy-template-id",
        "marketplace_id": "ATVPDKIKX0DER"
      }
    ],
    "merchant_suggested_asin": [
      {
        "value": "B071KQCQZ5",
        "marketplace_id": "ATVPDKIKX0DER"
      }
    ],
    "purchasable_offer": [
      {
        "currency": "USD",
        "our_price": [
          {
            "schedule": [
              {
                "value_with_tax": 10
              }
            ]
          }
        ],
        "marketplace_id": "ATVPDKIKX0DER"
      }
    ]
  },
  "productType": "PRODUCT",
  "requirements": "LISTING_OFFER_ONLY"
}
```

Have you been able to get around this somehow?

**Comment by chapmanjw** [comment]
@achie27 The `getListingsItem` operation shows two different representations of your pricing data. The `attributes` data is essentially what you gave Amazon and reflects almost immediately after you write the data. However, once data is accepted, it can take time to federate into each of the applicable systems for the pricing to be "live". The `offers` data reflects what is currently live for your listing. This is analogous to editing your price on the Manage Inventory page and that not reflecting for a few minutes.

While processing/federation delays can occur, if you see this on a regular basis for your listings taking longer than 15 minutes or so, I would suggest contacting Developer Support. This will enable us to dive deeper into your specific data and identify what might be causing delays (if specific to your submissions or submission pattern).

**Reply by achie27** [reply]
Thanks for the insight @chapmanjw, it's recurring and happening only with the API - I'll create a case with support.
